.buildkite/cache-builder.yml

```
@@ -0,0 +1,39 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/bash-cache#add/cocoapods-support: ~
+    - automattic/git-s3-cache#v1.1.0:
+        bucket: "a8c-repo-mirrors"
+        repo: "wordpress-mobile/wordpress-ios/"
+  # Common environment values to use with the `env` key.
+  - &common_env
+    IMAGE_ID: xcode-12.5.1
+
+steps:
+
+  #################
+  # Build the CocoaPods Base Cache
+  #################
+  - label: ":cocoapods: Rebuild CocoaPods cache"
+    command: |
+      echo "--- :rubygems: Setting up Gems"
+      install_gems
+
+      echo "--- :cocoapods: Rebuilding Pod Cache"
+      cache_cocoapods
+    env:
+      IMAGE_ID: xcode-12.5.1
```

If the step beneath needs access to the Xcode toolchain, then I'd suggest using the `common_env` anchor defined above here:

```suggestion
    env: *common_env
```

Otherwise, I'd suggest removing the `common_env` definition, since this would be the only usage.

---

Yep good call – addressed this in b53bd9e962452335c1d9a9b0101b2e1913ff6090 – IMHO it's nice to have the header the same in both pipeline files, so I like the reference approach 🙂